src/core/js/app.js

```diff
@@ -203,6 +204,10 @@ require([
         }
     }
 
+    function removeInitialLoadingMessage() {
```

Comment:
given that this is only called once I'm not sure it's worth making it into a function?